apps/web/src/services/cost-explorer/budget/budget-detail/modules/budget-info/BudgetDetailInfo.vue

```diff
@@ -87,26 +124,51 @@ const getTargetLabel = (projects: ProjectReferenceMap) => {
         <p-pane-layout class="summary-card">
             <span class="summary-title">{{ $t('BILLING.COST_MANAGEMENT.BUDGET.DETAIL.TARGET') }}</span>
             <p v-if="!budgetPageState.loading"
-               class="summary-content"
+               class="summary-content target"
             >
+                <span v-if="budgetPageState.budgetData?.project_id"
+                      class="target-project-group"
+                >
+                    <span>
+                        {{ getTargetLabel(state.projects).group }}
+                    </span>
+                    <p-i name="ic_chevron-right-thin"
+                         width="0.75rem"
+                         height="0.75rem"
+                         :color="gray[400]"
+                    />
+                </span>
                 <p-link v-if="state.budgetData?.project_group_id || state.budgetData?.project_id"
                         :action-icon="ACTION_ICON.INTERNAL_LINK"
                         new-tab
+                        highlight
                         :to="referenceRouter(
                             (state.budgetData?.project_id || state.budgetData?.project_group_id) ?? '',
                             { resource_type: state.budgetData?.project_id ? 'identity.Project' : 'identity.ProjectGroup' })"
                 >
-                    {{ getTargetLabel(state.projects) }}
+                    {{ getTargetLabel(state.projects).name }}
                 </p-link>
             </p>
         </p-pane-layout>
         <p-pane-layout class="summary-card">
             <span class="summary-title">{{ $t('BILLING.COST_MANAGEMENT.BUDGET.DETAIL.PROVIDER') }}</span>
-            <p v-if="!budgetPageState.loading"
-               class="summary-content cost-type"
+            <div v-if="!budgetPageState.loading"
+                 ref="costTypeWrapperRef"
+                 class="cost-type-wrapper"
             >
-                <span class="cost-type-content">{{ state.processedProviderValue }}</span>
-            </p>
+                <span ref="costTypeRef"
+                      class="summary-content cost-type"
+                      :class="['summary-content', 'cost-type', {'is-view-all': !state.isTextTruncate}]"
+                >
+                    <span class="cost-type-content">{{ state.processedProviderValue }}</span>
+                </span>
+                <p-text-button v-if="state.isTextTruncate"
+                               style-type="highlight"
+                               @click="handleClickViewAll"
+                >
+                    {{ $t('BILLING.COST_MANAGEMENT.BUDGET.VIEW_ALL') }}
+                </p-text-button>
+            </div>
```

Comment:
Issue: Oh... I'm so sorry. I didn't know it was a design feature and I removed it.
Additionally, it seems to be displayed as a popover like this.
![image](https://github.com/cloudforet-io/console/assets/50662170/e1135ddb-1894-4761-9acb-09e15a4a2205)